tests/publiccloud/download_repos.pm

```diff
@@ -0,0 +1,49 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Download repositores from the internal server
+#
+# Maintainer: Pavel Dostal <pdostal@suse.cz>
+
+use base 'consoletest';
+use registration;
+use warnings;
+use testapi;
+use strict;
+use utils;
+
+sub run {
+    my ($self, $args) = @_;
+    select_console 'root-console';
+
+    my $domain = '';
+    my @repos  = split(/,/, get_var('INCIDENT_REPO'));
+    assert_script_run("mkdir ~/repos");
+    assert_script_run("cd ~/repos");
+    while (defined(my $maintrepo = shift @repos)) {
+        my @path = split(/\//, $maintrepo);
+        splice(@path, 0, 2);
+        $domain = $path[0];
+        my $parent = join('/', @path) . "/";
+        assert_script_run("wget --no-clobber -r -R 'robots.txt,*.ico,*.png,*.gif,*.css,*.js,*.htm*' --domains $domain --no-parent $parent $maintrepo", timeout => 600);
+        assert_script_run("find $parent");
```

Comment:
What about explicit uploading the  list, something like:
```
assert_script_run("find $parent > /tmp/my_repo_temp_file"); upload_logs('/tmp/my_repo_temp_file');
```

Reply:
It will happen overnight ;)